core/arch/arm/table_a32_unpred.c

```diff
@@ -298,11 +298,11 @@ const instr_info_t A32_ext_bit7[][2] = {
  */
 const instr_info_t A32_ext_bit19[][2] = {
   { /* 0 */
-    {EXT_SIMD8,         0xf2800000, "(ext simd8  0)", xx, xx, xx, xx, xx, no, x, 0},
-    {EXT_SIMD5,         0xf2880000, "(ext simd5  0)", xx, xx, xx, xx, xx, no, x, 0},
+    {EXT_SIMD8,         0xf2800010, "(ext simd8  0)", xx, xx, xx, xx, xx, no, x, 0},
```

Comment:
OK, it looks like all children have bit 4 set, but it doesn't really matter at this split point: just informative, nothing reads it.